targets/TARGET_STM/TARGET_STM32WL/TARGET_STM32WLE5xC/TOOLCHAIN_ARM/stm32wle5xc.sct

```diff
@@ -0,0 +1,57 @@
+#! armcc -E
```

Comment:
As you know, to align with other targets

Reply:
Done

Reply:
Done